packages/grid_client/docs/billing.md

```diff
@@ -0,0 +1,334 @@
+# Billing
+
+## Description
+
+A couple of examples explaining how the billing works on the threefold grid for node and rent contracts.
+
+Some of the used resources should be updated whenever you try to do these calculations, these resources are the TFT price and Cloud Units(SU, CU).
+
+## Current TFT Price
+
+TFT price can be retrieved either from the dashboard, through the price available in the header
+
+![image](https://user-images.githubusercontent.com/101194226/252134103-e7cce566-311d-4293-8e73-358d45b616d1.png)
```

Comment:
Please update the image.

![image](https://github.com/user-attachments/assets/72056c4d-68b4-44a4-9d3c-ff1dd3cee064)